style.scss

```diff
@@ -1,187 +1,187 @@
+    #burger {
+      visibility: hidden;
+    }
+    .logoNav {
+      width: 129px;
+      height: 138px;
     }
 
-
-    @media screen and (min-width: $medium) {
-
-        .logoNav{
-            position: absolute;
-            visibility: visible;
-            top: 34px;
-            left: 40.95px;
-            width: 51px;
-            height: 54px;
-            filter: invert(64%) sepia(26%) saturate(344%) hue-rotate(156deg) brightness(100%) contrast(84%);
-        }
-        #burger{
-            right: 50px;
-            top: 34px;
-        }
+    #navItems {
+      visibility: visible;
+      padding-top: 0px;
+      display: flex;
+      justify-content: flex-end;
+      gap: 66px;
+      padding-right: 114px;
+      border-bottom: 5px yellow solid;
+      margin-left: 1112px;
+
+      .navItem {
+        list-style: none;
+        padding-top: 60px;
+        padding-bottom: 38px;
       }
-    @media screen and (min-width: 1020px){
-        #burger{
-            visibility: hidden;
-        }
-        .logoNav{
-            width: 129px;
-            height: 138px;
-        }
-        
-        #navItems {
-            visibility: visible;
-            padding-top: 0px;
-            display: flex;
-            justify-content: flex-end;
-            gap: 66px;
-            padding-right: 114px;
-            border-bottom: 5px yellow solid;
-            margin-left: 1112px;
-            
-
-            .navItem{
-                list-style: none;
-                padding-top: 60px;
-                padding-bottom: 38px;
-            }
-        }
     }
+  }
 }
 
-.deletethis {
-    width: 100vw;
-    height: 100vh;
-    background-color: blue;
+.landingPage {
+  background-image: url("../img/jeans1.png");
+  height: 100vh;
+  width: 100vw;
+  position: relative;
+  background-size: cover;
+  background-repeat: no-repeat;
 }
 
-footer {
-    height: 353px;
-    background: #87AABC 0% 0% no-repeat padding-box;
-    opacity: 1;
-    padding: 47px 53px 81px 53px;
+.headingContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #fcdd3f;
+  position: absolute;
+  top: 220px;
+  width: 100%;
+  .blueRecycleHeading {
+    width: 80%;
+  }
+  .buttonContainer {
     display: flex;
-    flex-direction: column-reverse;
-    justify-content: center;
-    align-items: center;
-    box-sizing: border-box;
-}
-
-.footerNav {
-    list-style: none;
-    margin-bottom: 35px;
-    & li {
-        text-align: center;
-        font: normal normal 600 25px/33px Segoe UI Semibold;
-        letter-spacing: 0px;
-        color: #002B50;
-        opacity: 1;
-        margin-bottom: 17px;
-        &:last-child {
-            margin-bottom: 0;
-        }
+    flex-direction: column;
+  }
+  .shopBtn {
+    width: 203px;
+    margin-bottom: 21px;
+    border-radius: 43px;
+    height: 54px;
+    background-color: #fcdd3f;
+    border: none;
+    font: normal normal bold 600 22px/30px Segoe UI;
+    font-size: 28px;
```

Comment:
Consider setting font sizes throughout the document in `rem` or `em` to enhance accessibility. Example utility function [here](https://gist.github.com/garystorey/5920051#gistcomment-3203271).